pallets/capacity/src/lib.rs

```diff
@@ -174,8 +177,8 @@ pub mod pallet {
 		#[pallet::constant]
 		type StakingRewardsPastErasMax: Get<u32>;
 
-		// The StakingRewardsProvider used by this pallet in a given runtime
-		// type RewardsProvider: StakingRewardsProvider<Self>;
+		/// The StakingRewardsProvider used by this pallet in a given runtime
+		type RewardsProvider: StakingRewardsProvider<Self>;
```

Comment:
It seems like StakingRewardsProvider is implemented below. Why do we inject it here too?